src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    /**
+     * returns true if the session is closed within the past hour of calling this function
```

Comment:
True, but the method should not rely on how it is called in the current code. It can be written to ensure the said situation will never happen (i.e. include only one of the two boundaries for the comparison - for example, return false if the deadline is exactly now).